custom-properties/set_org_custom_properties.py

```diff
@@ -0,0 +1,152 @@
+import os
+import requests
+import json
+import logging
+
+# Setup logging
+logging.basicConfig(level=logging.INFO)
+
+# GitHub API base URL
+API_BASE = "https://api.github.com"
+
+# Get OAuth token from environment variable
+TOKEN = os.environ.get('OAUTH_TOKEN')
+if not TOKEN:
+    raise ValueError("OAUTH_TOKEN environment variable is not set")
+
+# Your organization name
+ORG_NAME = "hmcts-test"
+
+# Headers for API requests
+headers = {
+    "Authorization": f"Bearer {TOKEN}",
+    "Accept": "application/vnd.github+json"
+}
+
+def define_custom_property(org_name):
+    """
+    Define a custom property for the organization.
+    """
+    url = f"{API_BASE}/orgs/{org_name}/properties/schema/is_production"
+    data = {
+        "value_type": "true_false",
+        "required": False,
+        "default_value": "",
+        "description": "Indicates if the repository is in production",
+        "allowed_values": None,
+        "values_editable_by": "org_and_repo_actors"
+    }
+    response = requests.put(url, headers=headers, json=data, verify=True)
+    if response.status_code != 200:
+        logging.error(f"Failed to define custom property for {org_name}: {response.json().get('message', 'Unknown error')}")
+    response.raise_for_status()
+    return response.status_code
+
+def get_org_repos(org_name):
```

Comment:
Not sure why we want all repos in org, if we're passing in a list of accepted repos?

Reply:
Removing the get_org_repos and updating the script to only read from JSON file. Previous code was used as a test for all repos